docs/pages/Layout.md

```diff
@@ -218,7 +218,7 @@ In a real application, these would likely be links and buttons.
   color='white'
   bg='blue'>
   <Icon
-    name='hotel'
+    name='Hotels'
```

Comment:
these two are the ones that broke the docsite, since `hotel` doesn't exist (it is `Hotels`)